esp/bindings/http/platform/httpprot.hpp

```diff
@@ -90,6 +94,10 @@ class CHttpThread: public CEspProtocolThread
     ISecureSocketContext* m_ssctx;
     IPersistentHandler* m_persistentHandler = nullptr;
     bool m_shouldClose = false;
+    IHttpServerService* m_httpserver = nullptr;
+    bool m_socketReturned2 = false;
```

Comment:
Not a huge fan of the 2 here.  Maybe call the m_sockerReturned one "m_pooledSocketReturned"?  or whatever names give clarity.

Reply:
2 here is not in comparison to the pooled, but the parent class CEspProtocolThread. Basically there're 2 "socketReturned" variables, one is inherited from the parent, one is from CHttpThread itself. But I'll try to come up with a better name...